src/routes/safe/store/actions/createTransaction.js

```diff
@@ -99,38 +101,54 @@ const createTransaction = ({
 
   let txHash
   let tx
+  const txArgs = {
+    safeInstance,
+    to,
+    valueInWei,
+    txData,
+    operation,
+    nonce,
+    safeTxGas: 0,
+    baseGas: 0,
+    gasPrice: 0,
+    gasToken: ZERO_ADDRESS,
+    refundReceiver: ZERO_ADDRESS,
+    from,
+    sigs,
+  }
+
   try {
     if (isExecution) {
       tx = await getExecutionTransaction(
-        safeInstance,
-        to,
-        valueInWei,
-        txData,
-        operation,
-        nonce,
-        0,
-        0,
-        0,
-        ZERO_ADDRESS,
-        ZERO_ADDRESS,
-        from,
-        sigs,
+        txArgs.safeInstance,
```

Comment:
can you just spread the `txArgs` object

Reply:
how, I have to change the function signature for `getApprovalTransaction` and `getExecutionTransaction` to receive an object instead of independen arguments. Do you something else in mind?

Reply:
<img width="182" alt="Screenshot 2020-02-18 at 20 22 00" src="https://user-images.githubusercontent.com/16622558/74755302-6749a480-528c-11ea-9e6f-6176570fdee0.png">

Reply:
but with this amount of arguments I think receiving an object is the way to go honestly

Reply:
@mikheevm 

I just send a new commit, with this change. 

It's a critical part the of the app, I would really appreciate to help me to test it well.

Also, I had to change `processTransaction.js` file. And I noted that is almost identical to `createTransaction.js`. I do not understand well what the difference are but I'm pretty sure that both files can be merged.

Reply:
if you are agreed with these changes, I suggest to merge this PR once it's well tested and I will continue with the #501 ticket (the visual part) in another PR.